doc/cephadm/host-management.rst

```diff
@@ -228,6 +228,83 @@ create a new CRUSH host located in the specified hierarchy.
 
 See also :ref:`crush_map_default_types`.
 
+OS Tuning Profiles
+==================
+
+Cephadm supports managing os tuning profiles that allow users to apply a set of
+sysctl settings to a given set of hosts. To do this, first create a yaml spec
+of the following format
+
+.. code-block:: yaml
+
+    profile_name: 23-mon-host-profile
+    placement:
+      hosts:
+        - mon-host-01
+        - mon-host-02
+    settings:
+      fs.file-max: 1000000
+      vm.swappiness: '13'
+
+Then apply the tuned profile with::
+
+    ceph orch tuned-profile apply -i <tuned-profile-file-name>
+
+This profile will then be written to /etc/sysctl.d/ on each host matching the
+given placement and `sysctl --system` will be run on the host.
+
+.. note:: 
+
+  The file name the tuned profile will be written under is <profile-name>-cephadm-tuned-profile.conf.
+  Since the settings are applied in lexicographical order, you may want to set the profile name
+  for your spec in a way that is applied before or after other conf files that may exist.
+
```

Comment:
suggest: 

The file name the tuning profile will be written to is <profile-name>-cephadm-tuned-profile.conf.
  Since the settings are applied in lexicographical order, you may want to set the profile name
  for your spec in a way that is applied before or after other conf files that may exist.
  
  
  I'm a bit confused as to the filename that will be written.  Above I would think that "-i <tuned-profile-file-name>" would result in "/etc/sysctl.d/<tuned-profile-file-name>", but this seems to imply that a "-cephadm-tuned-profile.conf" suffix will be added.  Do we actually use the `profile-name` defined within the YAML file to generate the output hostname?  If so, I suggest that we be more explicit about that.

Reply:
@anthonyeleven I rewrote this section to try and make it a bit clearer. Let me know what you think.

Reply:
Better.  Thanks!